zcash_client_backend/src/scanning.rs

```diff
@@ -365,22 +365,57 @@ pub(crate) fn scan_block_with_runner<
             if m.sapling_commitment_tree_size == 0 {
                 None
```

Comment:
This will be `None` if we are after Sapling activation and haven't had any Sapling outputs yet. The `or_else` below will then keep the result at `None` which will report a `ScanError::TreeSizeUnknown`. That seems wrong; should it be this?
```suggestion
                Some(0)
```